testsuite/MDAnalysisTests/__init__.py

```diff
@@ -102,6 +102,9 @@
 import os
 import sys
 
+# To test duecredit in utils/test_duecredits.py
+os.environ['DUECREDIT_ENABLE']='yes'
```

Comment:
why not set it in travis directly. This editing of environment variables is unusual.

Reply:
We initially had it in travis. I asked for it to be moved inside the tests. My reasoning was that this should be part of the tests (as we enable it for all tests). Otherwise one needs to know that in order to run the tests you also have to set a specific environment variable. It seemed more brittle to me to split the information.

But I can be convinced otherwise.

Reply:
So we *always* want to test due-credit? Currently, there is no way of disabling the tests even with an environment variable. I would then skip the whole environment variable thing because it just introduced extra steps that are not necessary.